Automattic/wp-calypso
# Deduplicate @wordpress packages

The last upgrade seems to have left behind a number of fixable duplicates.

#### Changes proposed in this Pull Request

* Deduplicate `@wordpress` packages

#### Testing instructions

* Ensure that Calypso builds successfully.
* Ensure that `@wordpress`-using sections, like `/new`, continue to work correctly.
